x-pack/plugins/observability_solution/apm/public/utils/get_timestamp.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function getTimestamp(timestamp?: string, fallbackTimestamp?: number): number {
+  if (!fallbackTimestamp && timestamp) {
```

Comment:
In case `timestamp` and `fallbackTimestamp` are both `true` then `fallbackTimestamp` is returned. I think it would be better to rely on `timestamp` in that scenario, wdyt?

maybe an opposite condition?
```
if (!timestamp && fallbackTimestamp) {
  return  fallbackTimestamp;
}

return new Date(timestamp).getTime() * 1000 || 0
```

Reply:
That would make sense, let's see what others think and I can implement it if everyone agrees

Reply:
changed to `timestamp` and `timestampUs` as the order of the params should always be the same